src/java/htsjdk/samtools/SAMRecord.java

```diff
@@ -1179,6 +1181,15 @@ protected SAMBinaryTagAndValue getBinaryAttributes() {
         return mAttributes;
     }
 
+    @Override
```

Comment:
Comment about what this concretely returns.  Does the clipping affect mapping, is it the span over the genome, what happens for unmapped reads etc.?